lib/grr.js

```diff
@@ -1639,6 +1656,8 @@ function stepReviewCRs(arg, cb) {
             arg.log.trace({err: pipelineErr}, 'pipelineErr');
             cb(pipelineErr);
         }
+        cb();
```

Comment:
This results in `cb()` being called twice. This pipeline finish could be replaced with:

```
}, function finish(pipelineErr) {
    cb(pipelineErr);
}
```

Reply:
Yep, incorporated this fix as part of 
commit beca7c1e1e3c89382500e6c4487153ecdf855630
Author: Tim Foster <tim.foster@joyent.com>
Date:   Wed Aug 14 14:20:38 2019 +0100

    Trent's patch to add more spacing between diffs when showing reviews